kubernetes_deploy/staging/service.yml

```diff
@@ -7,6 +7,6 @@ spec:
   ports:
   - port: 80
     name: http
-    targetPort: 80
+    targetPort: 8000
   selector:
-    app: laa-legal-adviser-api-app
+    app: laa-legal-adviser-api
```

Comment:
This file's change seems a bit out of place here, do we need it? I see two commits refer to "Correct targetPort in service definition" and "Correct app name in service definition". Should these changes be applied separately?

Reply:
They're both required in order for the service to serve the app. They could be made into their own PR but I'm not convinced there's value there to be honest

Reply:
It's the conceptual building of the history I was worried about; I thought that changes to this file would show up belonging to this pull request and that may be confusing as this pull request is about "Upload static files via jobs during deployment".

I put a lot of value in creating a history that always leads to a clear answer to "why did this have to change" and "why here".

_not a blocker_ though

Reply:
This PR's about getting the app pods set up properly, though; although in practice yeah that's mostly been the collectstatic stuff. So the choices are move that out, rename the PR, make another ticket and PR, or leave as is.

Reply:
Not a blocker, so your call. I think going forward in this codebase it's easy to split up functional changes into separate sets.